test/unittest/blas3/blas3_trsm_test.cpp

```diff
@@ -84,7 +92,12 @@ void run_test(const combination_t<scalar_t> combi) {
       blas::helper::copy_to_host<scalar_t>(q, b_gpu, B.data(), B.size());
   sb_handle.wait(event);
 
-  bool isAlmostEqual = utils::compare_vectors(cpu_B, B);
+  bool isAlmostEqual = utils::compare_vectors(
+      cpu_B, B, std::cerr, "",
```

Comment:
```suggestion
      cpu_B, B, std::cerr, "\n",
```
You need to pass `"\n"` as an argument to `compare_vector` otherwise the error output will be malformed.

*Apply this to all the occurrences of `compare_vectors` in which the 'error multiplier' argument is present.*